web/app/themes/mitlib-parent/footer.php

```diff
@@ -15,15 +15,10 @@
 
 		<div class="footer-info-institute">
 			<a class="link-logo-mit" href="https://www.mit.edu">
-				<span class="sr">MIT</span>
-				<svg version="1.1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" width="54" height="28" viewBox="0 0 54 28" enable-background="new 0 0 54 28" xml:space="preserve" class="logo-mit"><rect x="28.9" y="8.9" width="5.8" height="19.1" class="color"/><rect width="5.8" height="28"/><rect x="9.6" width="5.8" height="18.8"/><rect x="19.3" width="5.8" height="28"/><rect x="38.5" y="8.9" width="5.8" height="19.1"/><rect x="38.8" width="15.2" height="5.6"/><rect x="28.9" width="5.8" height="5.6"/></svg>
-			</a><!-- End MIT Logo -->
+				<img src="https://cdn.libraries.mit.edu/files/branding/local/mit_lockup_std-three-line_rgb_white.svg" alt="MIT" width="152">
```

Comment:
Should the width be 150? I'm not going to quibble over 2px; just want to make sure I have the right values in Springshare.

Reply:
In WordPress, the logo I'm matching ends up being 152px wide - but I think for Springshare it is probably fine to go with 150? I haven't checked the exact size there, and I'm also not sure that 2px difference would be flagged by almost anyone.